src/main/java/stirling/software/SPDF/config/EndpointConfiguration.java

```diff
@@ -76,154 +78,153 @@ public void disableGroup(String group) {
-        addEndpointToGroup("Python", "url-to-pdf");
-
-        // openCV
-        addEndpointToGroup("OpenCV", "extract-image-scans");
-        addEndpointToGroup("OpenCV", REMOVE_BLANKS);
-
-        // LibreOffice
-        addEndpointToGroup("LibreOffice", "repair");
-        addEndpointToGroup("LibreOffice", "file-to-pdf");
-        addEndpointToGroup("LibreOffice", "xlsx-to-pdf");
-        addEndpointToGroup("LibreOffice", "pdf-to-word");
-        addEndpointToGroup("LibreOffice", "pdf-to-presentation");
-        addEndpointToGroup("LibreOffice", "pdf-to-text");
-        addEndpointToGroup("LibreOffice", "pdf-to-html");
-        addEndpointToGroup("LibreOffice", "pdf-to-xml");
-
-        // OCRmyPDF
-        addEndpointToGroup("OCRmyPDF", "compress-pdf");
-        addEndpointToGroup("OCRmyPDF", "pdf-to-pdfa");
-        addEndpointToGroup("OCRmyPDF", "ocr-pdf");
-
-        // Java
-        addEndpointToGroup("Java", "merge-pdfs");
-        addEndpointToGroup("Java", "remove-pages");
-        addEndpointToGroup("Java", "split-pdfs");
-        addEndpointToGroup("Java", "pdf-organizer");
-        addEndpointToGroup("Java", "rotate-pdf");
-        addEndpointToGroup("Java", "pdf-to-img");
-        addEndpointToGroup("Java", "img-to-pdf");
-        addEndpointToGroup("Java", "add-password");
-        addEndpointToGroup("Java", "remove-password");
-        addEndpointToGroup("Java", "change-permissions");
-        addEndpointToGroup("Java", "add-watermark");
-        addEndpointToGroup("Java", "add-image");
-        addEndpointToGroup("Java", "extract-images");
-        addEndpointToGroup("Java", "change-metadata");
-        addEndpointToGroup("Java", "cert-sign");
-        addEndpointToGroup("Java", "multi-page-layout");
-        addEndpointToGroup("Java", "scale-pages");
-        addEndpointToGroup("Java", "add-page-numbers");
-        addEndpointToGroup("Java", "auto-rename");
-        addEndpointToGroup("Java", "auto-split-pdf");
-        addEndpointToGroup("Java", "sanitize-pdf");
-        addEndpointToGroup("Java", "crop");
-        addEndpointToGroup("Java", "get-info-on-pdf");
-        addEndpointToGroup("Java", "extract-page");
-        addEndpointToGroup("Java", "pdf-to-single-page");
-        addEndpointToGroup("Java", "markdown-to-pdf");
-        addEndpointToGroup("Java", "show-javascript");
-        addEndpointToGroup("Java", "auto-redact");
-        addEndpointToGroup("Java", "pdf-to-csv");
-        addEndpointToGroup("Java", "split-by-size-or-count");
-        addEndpointToGroup("Java", "overlay-pdf");
-        addEndpointToGroup("Java", "split-pdf-by-sections");
-        addEndpointToGroup("Java", REMOVE_BLANKS);
-
-        // Javascript
-        addEndpointToGroup("Javascript", "pdf-organizer");
-        addEndpointToGroup("Javascript", "sign");
-        addEndpointToGroup("Javascript", "compare");
-        addEndpointToGroup("Javascript", "adjust-contrast");
+    private void initialize() {
+        addEndpointsToGroup(
```

Comment:
This is cleaner code but I dislike the readability of it, the group name looks like a endpoint here
Perhaps dont use elispe operator and just new a new String[] for each?